packages/gatsby-theme-docs/src/modules/ai-assistant/components/chat-modal-css-components.js

```diff
@@ -117,6 +118,9 @@ export const SideTopContainer = styled.div`
   flex-grow: 1;
   overflow-y: auto;
   padding: 0px ${designSystem.dimensions.spacings.m};
+  border: 1px solid ${designSystem.colors.light.borderPrimary};
+  background-color: ${designSystem.colors.light.surfacePrimary};
+  border-radius: 4px;
```

Comment:
Not convinced to use a token whose name refers specifically to the beta flat for other purposes just because it's the same value.   Would lead to hidden side effects if someone later wants to change the beta flag and in good trust believes the naming.